setup.py

```diff
@@ -26,6 +26,7 @@
         'django-oauth-toolkit>=0.9.0',
         'python-social-auth>=0.2.2',
         'shortuuid>=0.4.3',
+        'django-shortuuidfield==0.1.3',
```

Comment:
As `locarise_drf_oauth2_support` is a third dependency, it's better to let the main project to install a new version of `django-shortuuidfield` with `'django-shortuuidfield>=0.1.3'`